beacon_node/lighthouse_network/gossipsub/src/behaviour.rs

```diff
@@ -1837,7 +1837,7 @@ where
 
         // Record the received message with the metrics
         if let Some(metrics) = self.metrics.as_mut() {
-            metrics.msg_recvd(&message.topic);
+            metrics.msg_recvd(&message.topic, message.get_size());
```

Comment:
Here you use `get_size()` and above we use `raw_protobuf_len` what motivates the difference?

Reply:
I think we should use `raw_protobuf_len` to obtain the byte length of received message. `Message` is transformed data from `RawMessage` and is missing some fields compared to `RawMessage`, so `message.get_size()` returns a smaller byte size than the actually received bytes.

Reply:
yeah the `get_size` function is unrequired we can just use `raw_protobuf_len`